frontend/src/styles/design-tokens/global/spacing.css

```
@@ -1,36 +1,78 @@
+  --padding-right-xsmall: var(--padding-xsmall);
+  --padding-bottom-xsmall: var(--padding-xsmall);
+
+   /* Padding - small (8px) */
+  --padding-small: 8px;
+  --padding-left-small: var(--padding-small);
+  --padding-top-small: var(--padding-small);
+  --padding-right-small: var(--padding-small);
+  --padding-bottom-small: var(--padding-small);
+
+  /* Padding - Small (8px) */
+  --padding-small: 8px;
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--padding-small is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)